pkgs/by-name/gi/git-warp-time/package.nix

```diff
@@ -0,0 +1,78 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+
+  # nativeBuildInputs
+  zstd,
+  pkg-config,
+  jq,
+  cargo,
+  rustc,
+  rustPlatform,
+
+  # buildInputs
+  libgit2,
+  typos,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "git-warp-time";
+  version = "0.8.4";
+
+  src = fetchurl {
+    url = "https://github.com/alerque/git-warp-time/releases/download/v${finalAttrs.version}/git-warp-time-${finalAttrs.version}.tar.zst";
+    sha256 = "sha256-Xh30nA77cJ7+UfKlIslnyD+93AtnQ+8P3sCFsG0DAUk=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit (finalAttrs) src;
+    nativeBuildInputs = [ zstd ];
```

Comment:
Ah ha! This is the missing bit of incantation I couldn't sus out. I do still wonder why it works locally for me but not in CI though. Is that because my local cache had the successful extract from the ZIP already so it skip the extraction phase?

Reply:
> Ah ha! This is the missing bit of incantation I couldn't sus out. I do still wonder why it works locally for me but not in CI though. Is that because my local cache had the successful extract from the ZIP already so it skip the extraction phase?

Oh I get it now.. Yea let's see the error it will give and report an issue at https://github.com/NixOS/ofborg/issues/ . I'm pretty sure that Hydra won't fail either way.